invenio/legacy/inveniocfg.py

```diff
@@ -832,9 +832,9 @@ def _detect_ip_address():
     """
     try:
         s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-        s.connect(('invenio-software.org', 0))
```

Comment:
> Perhaps the issue is mentioning google in the code?

`invenio-software.org` should be up and running if you want to install demosite hence it should be fine for this case too.